templates/artist/includes/artist_detail.html

```diff
@@ -0,0 +1,6 @@
+{% load thumbnail %}
```

Comment:
Ohh. I didn't mean to do that. 

The other pull request was for thumbnails. Might have accidentally added it to the wrong one